filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilderFactory.scala

```diff
@@ -0,0 +1,20 @@
+package cromwell.filesystems.blob
+
+import cromwell.core.path.PathBuilderFactory
+import akka.actor.ActorSystem
+import cromwell.core.WorkflowOptions
+import cromwell.filesystems.blob.BlobPathBuilder
+import scala.concurrent.{ExecutionContext, Future}
+import com.typesafe.config.Config
+import com.azure.core.credential.AzureSasCredential
+
+final case class BlobPathBuilderFactory(globalConfig: Config, instanceConfig: Config) extends PathBuilderFactory {
+  override def withOptions(options: WorkflowOptions)(implicit as: ActorSystem, ec: ExecutionContext): Future[BlobPathBuilder] = {
+    val sasToken: String = instanceConfig.getString("sasToken")
+    val container: String = instanceConfig.getString("store")
+    val storageAccount: String = instanceConfig.getString("storageAccount")
```

Comment:
It looks like the config being used here is different from what was added to `reference.conf` above. Maybe remove the `reference.conf` section? As discussed, not important to get the config right for this PR, but might as well removed things that are inconsistent and unused.